Server/src/redis/MsgBusImpl_redis.cpp

```diff
@@ -0,0 +1,243 @@
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+
+#include <cinttypes>
+
+#include <netdb.h>
+#include <unistd.h>
+
+#include <thread>
+#include <arpa/inet.h>
+
+#include "MsgBusImpl_redis.h"
+#include "RedisManager.h"
+
+
+
+
+
```

Comment:
Could you follow code styles in nearby code files? No extra empty lines.

Reply:
fixed.